gbpservice/nfp/common/constants.py

```diff
@@ -65,6 +73,7 @@
     CUSTOM_JSON: 'custom_json'}
 
 LOADBALANCER_RPC_API_VERSION = "2.0"
+LOADBALANCERV2_RPC_API_VERSION = "1.0"
```

Comment:
Is LBaaS V2 plugin restricts this API version? Better to use 2.0 right? If the restricts this, then we can retain this as there will not be a way to use 2.0 in that case.